requirements/edx/base.txt

```diff
@@ -66,7 +66,11 @@ cryptography==2.5
 cssutils==1.0.2           # via pynliner
 ddt==1.2.0
 decorator==4.3.2          # via dogapi, pycontracts
+<<<<<<< HEAD
```

Comment:
Need to fix some merge conflicts here (or just rerun `make upgrade` after rebasing)

Reply:
Grrr sorry about that